packages/block-library/src/breadcrumbs/index.php

```diff
@@ -159,6 +159,21 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 		array_pop( $breadcrumb_items );
 	}
 
+	/**
+	 * Filters the breadcrumb items array before rendering.
+	 *
+	 * Allows developers to modify, add, or remove breadcrumb items.
+	 *
+	 * @since 7.0.0
+	 *
+	 * @param array    $breadcrumb_items Array of breadcrumb item data. Each item is an array with:
+	 *                                   - 'label' (string) The breadcrumb text
+	 *                                   - 'url' (string, optional) The breadcrumb link URL
+	 *                                   - 'allow_html' (bool, optional) Whether to allow HTML in label
```

Comment:
Let's elaborate a bit in this prop and mention that with `allow_html` it will still pass through `wp_kses_post`.